spec/data-spec.js

```diff
@@ -185,6 +189,13 @@ describe("Data", () => {
 				expect(+circle.attr("cx")).to.be.closeTo(expectedCx[i], 0);
 				expect(+circle.attr("cy")).to.be.closeTo(expectedCy[556][i], 1);
 			});
+
+			d3SelectAll(".bb-circles-778-889 .bb-circle").each(function (d, i) {
+				const circle = d3Select(this);
+
+				expect(+circle.attr("cx")).to.be.closeTo(expectedCx[i], 0);
+				expect(+circle.attr("cy")).to.be.closeTo(expectedCy["778.889"][i], 0);
```

Comment:
Test log: `expected 347.21598639455783 to be close to 347 +/- 0`
Try increasing delta value. It happens because of rendering env differences, so try give some gaps on test cases.

It'll be better run local test first, if you can to avoid multiple commits :)